src/layout/Header/PngBalanceContent.tsx

```diff
@@ -141,11 +141,11 @@ export default function PngBalanceContent({ setShowPngBalanceModal }: { setShowP
         <CardSection gap="sm">
           <AutoColumn gap="md">
             <RowBetween>
-              <TYPE.white color="white">{t('header.pngPrice')}</TYPE.white>
+              <TYPE.white color="white">{t('header.pngPrice', { symbol: CHAINS[chainId].png_symbol })}</TYPE.white>
```

Comment:
this is not using `getPngSymbol`. please check in entire PR, I have seen some places where it is still not using `getPngSymbol`

Reply:
Corrected it.